8.x/theme-development/galleries.md

```diff
@@ -1,3 +1,138 @@
 ## Template galleries
 
-[[toc]]
\ No newline at end of file
+[[toc]]
+
+In the Larammerce Template Engine, any section that is repetitive and the website admin wants to create a list of content and edit it uses the gallery.
+
+For example, the image below is the faq page. On this page, there is a fixed section of questions and answers but repetitive.
+![faq-page.png](/faq-page.png)
+
+Let's take a look at how the gallery is made.
+
+First, one of the Larammerce template projects is reviewed, open the .blade file named: `public/views/faq.blade.php`:
+
+```html
+  <div class="top-line-content" hct-gallery="faqs" hct-title='پرسش وپاسخ ها' hct-max-entry="7" hct-random-select>
+```
+In this tag there are attributes, each of which is described:
+```html
+hct-gallery="faqs"
+```
+This attribute is to say that there is a gallery called faq. To name this attribute, the rules for variables must be observed.
+- A variable name must start with a letter or the underscore character.
+- A variable name cannot start with a number.
+- A variable name can only contain alpha-numeric characters and underscores (A-z, 0-9, and _ )
+
+```html
+hct-title='پرسش وپاسخ ها'
+```
+**hct-title** show the title of this gallery.
+
+```html
+hct-max-entry="7"
+```
+The gallery shows only seven of the contents created by the website admin.This value can be changed as desired.
+
+```html
+hct-random-select
+```
+That is, the order is random, if not set, in order of priority.
+
+There is an **ul** tag in the **div** tag, it must have a hidden class.
+
+```html
+<ul class="hidden-xl hidden-lg hidden-md hidden-sm hidden-xs" hct-gallery-fields>
+                <li hct-gallery-field="question" hct-title="سوال"></li>
+                <li hct-gallery-field="description" hct-title="جواب"></li>
+            </ul>
+```
+**hct-gallery-fields** specifies that each gallery item related to the faq gallery has fields called questions and answers.
+
+This table is an example to understand the difference between an item and an gallery item.
+
+gallery|gallery item
```

Comment:
It's not necessary to use a table to elucidate the meaning, you can tell:
For example, if it's going to be used to manage a slider of photos on the website, the slider itself would be the `gallery` and each slide (containing its photo, title, link, etc.) goes to be the `gallery item`.